### PR TITLE
Added Layer5 Writing Program to Community Handbook

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
- Added Layer5 Writing Program to Community Handbook

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit c7886e0f7c33dbc3c906295cc7946c7d4aa80aac at: https://62b253ae3ef5f300a43dd35d--layer5.netlify.app

File: src/pages/community/handbook/learn-layer5.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const CommunityLearn = () => (
   <ThemeProvider theme={theme}>
     <Layout>
       <GlobalStyle />
-      <SEO title="Learn Layer5" description="The purpose of the handbook is to provide an overview of the Layer5 community" />
+      <SEO title="Learn Layer5" description="We have put together good learning materials and resources to guide you through learning about cloud native infrastucture." />
```
Comment (leecalcote) [inline src/pages/community/handbook/learn-layer5.js:18]: Better

File: src/pages/community/handbook/mentorship-programs.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const Mentorship = () => (
   <ThemeProvider theme={theme}>
     <Layout>
       <GlobalStyle />
-      <SEO title="Mentorship Programs" description="The purpose of the handbook is to provide an overview of the Layer5 community" />
+      <SEO title="Mentorship Programs" description="As an open-source organization and a community, we participate in different mentorships programs." />
```
Comment (leecalcote) [inline src/pages/community/handbook/mentorship-programs.js:18]: Good of you to update these.

**Comment by leecalcote** [comment]
Good!

**Pull request merged**